### ISSUE TITLE
"Install in 5 minutes" Results in TypeError

### ISSUE THREAD

**Issue by mlinds** [geopython/pygeoapi#1335]
**Description**
I am getting a Python TypeError with the message "Field() missing 1 required positional argument: 'default'"

**Steps to Reproduce**
I followed the "Install in 5 minutes" instructions found [here](https://pygeoapi.io/).

```shell
python3 -m venv pygeoapi
cd pygeoapi
 . bin/activate
git clone https://github.com/geopython/pygeoapi.git
cd pygeoapi
pip3 install -r requirements.txt
python3 setup.py install
cp pygeoapi-config.yml example-config.yml
export PYGEOAPI_CONFIG=example-config.yml
export PYGEOAPI_OPENAPI=example-openapi.yml
pygeoapi openapi generate $PYGEOAPI_CONFIG --output-file $PYGEOAPI_OPENAPI
```

**Expected behavior**
The openAPI file should be generated

**Screenshots/Tracebacks**

The error occurs on the step `pygeoapi openapi generate $PYGEOAPI_CONFIG --output-file $PYGEOAPI_OPENAPI`
```
Traceback (most recent call last):
  File "/home/mlinds/miniconda3/bin/pygeoapi", line 5, in <module>
    from pygeoapi import cli
  File "/home/mlinds/miniconda3/lib/python3.10/site-packages/pygeoapi/__init__.py", line 40, in <module>
    from pygeoapi.config import config
  File "/home/mlinds/miniconda3/lib/python3.10/site-packages/pygeoapi/config.py", line 36, in <module>
    from pygeoapi.util import to_json, yaml_load
  File "/home/mlinds/miniconda3/lib/python3.10/site-packages/pygeoapi/util.py", line 73, in <module>
    from pygeoapi.models import config as config_models
  File "/home/mlinds/miniconda3/lib/python3.10/site-packages/pygeoapi/models/config.py", line 35, in <module>
    class APIRules(BaseModel):
  File "/home/mlinds/miniconda3/lib/python3.10/site-packages/pygeoapi/models/config.py", line 37, in APIRules
    api_version: str = Field(regex=r'^\d+\.\d+\..+$',
TypeError: Field() missing 1 required positional argument: 'default'
```

**Environment**
 - OS: Ubuntu 22.04.1 LTS (On WSL2
 - Python version: Python 3.10.8
 - pygeoapi version: latest from master branch

**Comment by hiteshbedre** [comment]
@mlinds Until I get feedback on my PR you can try with my version by cloning `https://github.com/hiteshbedre/pygeoapi`

**Comment by mlinds** [comment]
> @mlinds Until I get feedback on my PR you can try with my version by cloning `https://github.com/hiteshbedre/pygeoapi`

Hi @hiteshbedre! Thanks for your quick response. However, I realized that the error was not actually a bug in the codebase of pygeoapi, but rather due to weird interaction between conda and pip venvs; my `which python` was showing the venv but when I called `$ pygeoapi .....` it was not using the local install I just clone from github, but another version that I guess was floating around in my conda environment.
